cli/src/cli/scaffold/rust.rs

```diff
@@ -107,8 +114,14 @@ impl RustScaffold {
         let mut contents = String::new();
         cargo_file.read_to_string(&mut contents)?;
 
+        let template = match self.macro_style {
+            HdkMacroStyle::Declerative => include_str!("./rust/Cargo.template.toml"),
+            HdkMacroStyle::Procedural => include_str!("./rust-proc-macro/Cargo.template.toml"),
+        };
```

Comment:
Is this path resolution style windows compatible (and do we care) ?

Reply:
From a bit of research include_str!:

> Includes a utf8-encoded file as a string. The file is located relative to the current file. (similarly to how modules are found)

This means I was able to drop the relative path specifier. This should work fine on windows. It has been in the CLI this way since the beginning.